### ISSUE TITLE
[ALPSCore] [alps-gf-c++11] can't compile with gcc 4.7.2

### ISSUE THREAD

**Issue by iskakoff** [ALPSCore/ALPSCore#453]
gcc-4.7.2 can't construct std::array using list initialization.

**Comment by krivenko** [comment]
Strictly speaking, gcc 4.7 is not C++11-conformant.

https://gcc.gnu.org/gcc-4.8/cxx0x_status.html
```
As of GCC 4.8.1, GCC's C++11 mode implements all of the major features of the C++11 standard produced by the ISO C++ committee.
```
In practice, I would not recommend to use anything older than gcc 4.8.4, which had most of major c++11 bugs ironed out.

**Comment by iskakoff** [comment]
@krivenko yes, but Emanuel has fedora 17 build server for some reason, and it has gcc-4.7.2. So far it was the only issue, I guess

**Comment by iskakoff** [comment]
fixed by #454
